### PR TITLE
Simplify WPT runtime

### PR DESCRIPTION
**Pull request by kainino0x**
Tested manually on: https://chromium-review.googlesource.com/c/chromium/src/+/2528311

### CODE REVIEW

**Comment by kainino0x** [comment]
@austinEng PTAL

File: src/common/runtime/wpt.ts [hunk 1]
```diff
@@ -1,63 +1,59 @@
+// Implements the wpt-embedded test runner (see also: wpt/cts.html).
+
 import { DefaultTestFileLoader } from '../framework/file_loader.js';
 import { Logger } from '../framework/logging/logger.js';
 import { parseQuery } from '../framework/query/parseQuery.js';
-import { TestTreeLeaf } from '../framework/tree.js';
-import { AsyncMutex } from '../framework/util/async_mutex.js';
```
Comment (austinEng) [inline src/common/runtime/wpt.ts:5]: Is this the only usage of `AsyncMutex`? Can we remove it?

**Review by austinEng — APPROVED** [review]
Nice :)

File: src/common/runtime/wpt.ts (hunk 1 above)
Comment (kainino0x) [reply src/common/runtime/wpt.ts:5]: I think it is, and I was thinking about removing it but forgot. May as well do it now.

**Pull request merged**